content/en/certificate.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Certificate
+status: Feedback Appreciated
+category: technology
+---
+
+
+## What it is
+
+A certificate — also often referred to as a digital certificate, public key certificate, or SSL certificate — is a technology used to verify the identity of an entity on the internet. 
+An entity can be the sender of an email or a website. Additionally, certificates encrypt communication between entities on the internet.
+
+
+## Problem it addresses
+
+Communication over the internet is unencrypted by default. 
+Consequently, any communication can potentially be intercepted, compromising sensitive information like usernames and passwords. 
+Think about your favorite coffee shop or a hotel lobby, where you connect to the available wifi router. 
+Someone could get between your device and the wifi router and get hold of your credentials or read your emails and chats.
+
+
+## How it helps
+
+Certificates are a technology that, on the one side, verifies that communication partners are who they say they are — 
+for example, an email sender or a website that you visit on your browser. 
+Modern email clients will notify you if a sender's identity is correct, as will the browsers (notice the little lock in front of the URL). 
+A certificate's trust derives from the certificate authority, often abbreviated as CA, that issues the certificate, guaranteeing that the certificate holder's identity is correct.
```

Comment:
I think this detail can be omitted. It adds complexity to the definition in a way that isn't necessarily useful. 
```suggestion
```